DoxygenRule.cmake

```diff
@@ -186,11 +186,17 @@ if(GIT_DOCUMENTATION_REPO)
       COMMENT "Copying ${PROJECT_NAME} API documentation to ${GIT_DOCUMENTATION_DIR}"
       DEPENDS ${PROJECT_NAME}-doxygen VERBATIM)
   else()
+    # Having a command while force this target to run always when invoked. That
+    # makes the Makefile to print the comment. The text will be also visible
+    # in ninja.
     add_custom_target(${PROJECT_NAME}-doxycopy
+      COMMAND ${CMAKE_COMMAND} -E echo ""
       COMMENT "doxycopy target not available, missing ${_GIT_DOC_SRC_DIR}")
```

Comment:
Not sure. It looks nicer like this with the Makefile generator because it stands out with blue ink.